index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>HANGMAN</title>
+    <meta name="description" content="hangman">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Coiny&amp;subset=latin-ext" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class = 'container'>
+      <div class = 'header'>
+        <h1 class = 'title'>HANGMAN</h1>
+        <button class = 'btn'>START GAME</button>
+      </div>
+      <div class = 'game-wrapper'>
+        <div class = 'image-container'>
+          <img class = 'image' src="img/s0.jpg">
+        </div>
+        <div class = 'letter-container a border-bottom border-right'><p>A</p></div>
```

Comment:
empty line before it, but there is no need to add lines after each of the letter. 

You could consider craetion of t hose divs in JavaScript using forEach. In example:
```
['a', 'b', 'c' ....].forEach(letter => {
 const letter = document.createElement('div`). 
 ...handle adding classes, appending `p` element, text, etc, appending completed html element. 
})
```
It would be great enhancement.

Reply:
yes it will be created with js :) i just wanted to start this way ;)

Reply:
so for this PR let's leave as is